src/index.js

```diff
@@ -51,17 +51,35 @@ class App {
     this.container.appendChild(addTask);
     this.container.appendChild(this.taskListContainer);
     this.container.appendChild(buttonDeleteCompletedTasks);
+
+    this.taskList = this.getTaskListFromLocalStorage();
```

Comment:
assignment not needed this.getTaskListFromLocalStorage() would be enough

Reply:
done